workflow/sample_files/base-bldgtype-multifamily-adjacent-to-multifamily-buffer-space.xml

```diff
@@ -126,7 +126,7 @@
             </InteriorFinish>
             <Insulation>
               <SystemIdentifier id='Wall2Insulation'/>
-              <AssemblyEffectiveRValue>23.0</AssemblyEffectiveRValue>
+              <AssemblyEffectiveRValue>4.0</AssemblyEffectiveRValue>
```

Comment:
Why did this change?

Reply:
This is because tasks.rb assigns wall properties (and other surface types) based on index of walls (e.g., `hpxml.walls[-1]`) instead of searching for the wall based on its properties. Since the order of `hpxml.walls` may have changed, we need to update the index value.